src/components/dropdown/Dropdown.tsx

```diff
@@ -277,7 +298,7 @@ export class Dropdown<T = LabelValue> extends React.Component<
           tooltipCloseLabel={tooltipCloseLabel}
           showRequired={showRequired}
         />
-        <Select
+        <Dropdown 
```

Comment:
I am confused now, from where this is coming from?

Reply:
It will depend if you pass 'asyncOptions' or 'options' as prop

Reply:
sorry i don't get it : where are we using select now? 
thx

Reply:
ok got it, does it make sense to store "Dropdown" in the state? 
Also can we use another name because it's very confusing (it seems like we're rendering the same Dropdown component inside Dropdown component...)